### ISSUE TITLE
describe why a "stripped-down" version

### ISSUE THREAD

**Issue by drzraf** [leucos/ansible-ferm#1]
Hi,
I'm interested in this role, but since it originates from https://github.com/debops/ansible-ferm I'd be tempted to use upstream + ansible-galaxy.
Could you describe why a stripped-down version was needed and what benefits to expect from this?

thank you

**Comment by leucos** [comment]
Hi @drzraf 

First, I don't use galaxy. I prefer to fork an interesting role (or write from scratch), adapt, and use my version. This way, I can steer the role the way I want and do not depend on external decisions. I then use my roles as submodules in each playbook I write, so new commits on existing roles do not impact other playbooks using it.

I also have a fixed pattern for my roles: I usually don't have tasks in `main.yml`, I just use `include` in this file. I also simplified `when` clauses removing variables and features I don't need.

Few features have been added too, like automatic whitelisting hosts, automatic rule to allow ansible controller on ssh port, and handling IP forwarding in sysctl.

Dont get me wrong: @drybjed made a fantastic work on [debops](https://github.com/debops/), and he deserves tons of kudos for this. Debops roles are awesome and a gret reference to learn ansible. When I need to create a new role, my first move is to check his work. However, since debops roles are very generic, they tend to have too much features I don't need, and I like to keep things slim and simple, especially since I work with them on a daily basis.
